README.md

```diff
@@ -221,35 +223,39 @@ The following options are valid inside a part configuration:
   Explicit list of all values that will be iterated when bumping that specific
   part.
 
-  Example::
+  Example:
 
-    [bumpversion:part:release_name]
-    values =
-      witty-warthog
-      ridiculous-rat
-      marvelous-mantis
+  ```ini
+  [bumpversion:part:release_name]
+  values =
+    witty-warthog
+    ridiculous-rat
+    marvelous-mantis
+  ```
 
 #### `optional_value =`
   **default**: The first entry in `values =`.
 
   If the value of the part matches this value it is considered optional, i.e.
-  it's representation in a `--serialize` possibility is not required.
-
-  Example::
-
-    [bumpversion]
-    current_version = 1.alpha
-    parse = (?P<num>\d+)\.(?P<release>.*)
-    serialize =
-      {num}.{release}
-      {num}
-
-    [bumpversion:part:release]
-    optional_value = gamma
-    values =
-      alpha
-      beta
-      gamma
+  its representation in a `--serialize` possibility is not required.
+
+  Example:
+
+  ```ini
```

Comment:
There is no need to indent when it's a code block (and that actually makes copy-paste harder). This repeats for every block so I won't comment it everywhere.

Reply:
Solved in ae0d65d8a3d16f0ac5c32b87f71b9c7b1d493bff.